### PR TITLE
Tatiana belyukina

### PR DESCRIPTION
**Pull request by TatianaBelyukina**
<h1>Description</h1>

First we process the data:

delete this columns ('PassengerId', 'Ticket', 'Name', 'Cabin') because they don't 
have any useful information besides column 'Name' from where we get 'Age';

split  such columns as "Sex", "Embarked"  by values in order to get table with '0' and '1'
"Age", "Fare", "Parch", "SibSp"  split by intervals of values;

Training:
tried different models from sklearn. Compared accuracies and chose SVC 
with radial basis function that has ~82% accuracy, and 77.5 on kaggle  

<h2>How has this been tested?</h2>

- [ ] Executed code few times from start to end without crashing
- [ ] Only ran once
- [ ] Code have dedicated unit tests

### CODE REVIEW

**Comment by meanalexrin** [comment]
Please export IPython notebook to the .py file and include this .py file into the pull request 
https://stackoverflow.com/a/52885953

**Review by meanalexrin — CHANGES_REQUESTED** [review]
Please move you work to the `/[username]_code` location, as described in readme
https://github.com/rnd4u-org/2021-knu-cairl#development

File: TatianaBelyukina_code/task3/main.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from split_data import load_data
+from build_models import build_model
+
+x_train, y_train, x_test, y_test = load_data()
+model = build_model()
+
+model.fit(x_train, y_train)
```
Comment (meanalexrin) [inline TatianaBelyukina_code/task3/main.py:7]: Consider using `if name == "__main__":` to indicate entry point to program
It also may help you in future if you'll ever try to use `multiprocessing` library

**Review by meanalexrin — DISMISSED** [review]
LGTM :+1:
